### DISCUSSION TITLE
Full prismatic cell model?

### DISCUSSION THREAD

**Discussion by sandeepmam** [pybamm-team/PyBaMM#1711] [Q&A] [answered]
Is it possible to model a full prismatic cell in pybamm or pybamm can only do an electrochemical model? Are there any examples where can I refer to or does anyone know where can I find this kind of information?

**Comment by brosaplanella** [comment, marked as answer]
You can couple the electrochemical model to a lumped model which you could use for the prismatic cell. If you want models that resolve the geometry of the prismatic cell, then this is not immediately available in PyBaMM. For more information on the thermal models you can check [this notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/models/thermal-models.ipynb).

**Reply by sandeepmam** [reply]
Thank you for the information and the notebook. I hope it would give more insights.
